docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.md

```diff
@@ -12,13 +12,12 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.html
 ---
 
-## Introduction
-
 Manifests support registering event subscribers only in the dependency provider, a type of code class.
 
 Manifests fully support the registration of event subscribers in the collection. Restrictions to the order of the plugins in collection are **not supported**. New plugin will be added to the end of the collection.
```

Comment:
use italic to dray attention to smth: https://developers.google.com/style/text-formatting